winui/Shimmer/Customization.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Customization in WinUI Shimmer
+
+This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You can modify the following aspects of the Shimmer effect:
+
+## Fill
+
+The `Fill` property allows you to set the background color of the Shimmer view.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfShimmer Fill="#89CFF0" />
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfShimmer Shimmer = new SfShimmer();
+Shimmer.Fill = Color.FromRgba("#89CFF0");
+this.Content = Shimmer;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Fill customization in WinUI Shimmer](Shimmer_images/winui_shimmer_fill.gif)
+
+## Wave Color
+
+The `WaveColor` property lets you define the color of the shimmer wave.
+
```

Comment:
@GilbertmanojSF3971 The WaveColor property enables you to define the color of the shimmer wave.

Reply:
@Manivannan-E  Sir changed the content to The WaveColor property enables you to define the color of the shimmer wave in the PR.